backend/module/k8s/node.go

```diff
@@ -2,15 +2,22 @@ package k8s
 
 import (
 	"context"
-	"errors"
 
 	k8sapiv1 "github.com/lyft/clutch/backend/api/k8s/v1"
 )
 
 func (a *k8sAPI) DescribeNode(ctx context.Context, req *k8sapiv1.DescribeNodeRequest) (*k8sapiv1.DescribeNodeResponse, error) {
-	return nil, errors.New("not implemented")
+	node, err := a.k8s.DescribeNode(ctx, req.Clientset, req.Cluster, req.Name)
+	if err != nil {
+		return nil, err
+	}
+	return &k8sapiv1.DescribeNodeResponse{Node: node}, nil
 }
 
 func (a *k8sAPI) UpdateNode(ctx context.Context, req *k8sapiv1.UpdateNodeRequest) (*k8sapiv1.UpdateNodeResponse, error) {
-	return nil, errors.New("not implemented")
+	err := a.k8s.UpdateNode(ctx, req.Clientset, req.Cluster, req.Name, req.Unschedulable)
+	if err != nil {
+		return nil, err
+	}
+	return &k8sapiv1.UpdateNodeResponse{}, nil
```

Comment:
minor nit: should we return the updated node object here?

Reply:
we don't return the updated object for other k8s types, ex. pod https://github.com/lyft/clutch/blob/main/api/k8s/v1/k8s.proto#L458-L459, I implemented it this way to match the pattern. Either way is fine for me, I suspect I won't need the updated node object for the cordon node workflow though.

Reply:
Good point, for consistency let's keep as empty and I'll mention it in our next team meeting.